app.js

```diff
@@ -1,8 +1,11 @@
 
 import express from 'express';
+import swaggerUi from 'swagger-ui-express';
 import session from 'express-session';
 import cors from 'cors';
 import dotenv from 'dotenv';
+import swaggerDocument from './Learn Ground Api Documentation-Swagger20.json';
```

Comment:
Can you rename this json file? 

Could be lowercased all through with the spaces replaced with hyphens. Unless there are strict restrictions regarding the naming.

Reply:
ok. rightway